app/routes/api/applications.js

```diff
@@ -18,7 +18,7 @@ module.exports = [{
     },
     handler: async (request, h) => {
       const application = (await get(request.params.ref))
-      if (application.dataValues) {
+      if (application?.dataValues) {
```

Comment:
are we sure a caller isn't checking if the agreement/application has a claim or not? shouldn't be but guess it could be the case